integration/clean/clean_ami.go

```diff
@@ -8,27 +8,19 @@ package main
 
 import (
 	"context"
-	"log"
-	"time"
-
 	"github.com/aws/aws-sdk-go-v2/aws"
 	"github.com/aws/aws-sdk-go-v2/config"
 	"github.com/aws/aws-sdk-go-v2/service/ec2"
 	"github.com/aws/aws-sdk-go-v2/service/ec2/types"
 	smithyTime "github.com/aws/smithy-go/time"
+	"log"
+	"time"
 )
 
-func main() {
-	err := cleanAMI()
-	if err != nil {
-		log.Fatalf("errors cleaning %v", err)
-	}
-}
+var expirationDateAMI = time.Now().UTC().Add(keepDurationAMI)
 
 const daysToKeep = 60
-const keepDuration = -1 * time.Hour * 24 * time.Duration(daysToKeep)
-
-var expirationDate = time.Now().UTC().Add(keepDuration)
+const keepDurationAMI = -1 * time.Hour * 24 * time.Duration(daysToKeep)
```

Comment:
Does this name need to be changed? It's lowercase so AFAIK it shouldn't conflict with the duration used in the other file.